### DISCUSSION TITLE
Automatic Connection

### DISCUSSION THREAD

**Discussion by L-Reichardt** [foxglove/community#760] [Q&A] [answered]
Hello, I am trying to open foxglove with an automatic connection to ROS1, without manual user input. Is there a way to set a default, or call this through CLI?

**Comment by jtbandes** [comment, marked as answer]
Hi, you should be able to construct a link using these instructions: https://foxglove.dev/docs/studio/building-and-sharing-links#ros-1-native

**Reply by L-Reichardt** [reply]
Thank you very much, this is exactly what I was looking for.
